### ISSUE TITLE
Support sending code coverage reports to Codecov.io for DSC repositories

### ISSUE THREAD

**Issue by johlju** [gaelcolas/Sampler#99]
To be able to upload code coverage to Codecov.io the JaCoCo file that is returned by Pester needs to be parsed. This will only support modules that have the folder structure of the built module exactly the same as in the `source` folder. 

```
              $xml = Select-Xml -Path './output/testResults/CodeCov*.xml' -XPath '.'
              $nodes = $xml.Node.SelectNodes("/report/package");
              foreach ($node in $nodes)
              {
                  $node.SetAttribute('name', ($node.GetAttribute('name') -replace '\d+\.\d+\.\d+', 'source'))
              }
              $nodes = $xml.Node.SelectNodes("/report/package/class");
              foreach ($node in $nodes)
              {
                  $node.SetAttribute('name', ($node.GetAttribute('name') -replace '\d+\.\d+\.\d+', 'source'))
              }
              $nodes = $xml.Node.SelectNodes("/report/package/sourcefile/line");
              foreach ($node in $nodes)
              {
                  $node.SetAttribute('mb', 0)
                  $node.SetAttribute('cb', 0)
              }
              $xml.Node.Save("./output/testResults/JaCoCo_coverage.xml")
              $fileContent = Get-Content -Path './output/testResults/JaCoCo_coverage.xml' -Encoding 'Unicode' -Raw
              $fileContent | Out-File -FilePath './output/testResults/JaCoCo_coverage.xml' -Encoding 'ascii'
```

**Comment by johlju** [comment]
After the PR #101 is through I will update a DSC module repository to upload to Codecov.io. After that I will update the template accordingly which closes this issue.

**Comment by johlju** [comment]
This was resolved in PR #101.
